app/Services/Installer/MigrationService.php

```diff
@@ -68,6 +66,15 @@ public function runAllMigrations()
         Artisan::call('migrate');
         $output .= trim(Artisan::output());
 
+        $modules = Module::allEnabled();
+        foreach ($modules as $module) {
+            $module_path = $module->getPath().'/Database/migrations';
+            if (file_exists($module_path)) {
+                Artisan::call('module:migrate '.$module->getName());
+                $output .= trim(Artisan::output());
+            }
+        }
```

Comment:
Ok, we kinda ended up doing the same thing :) Nice

Reply:
Yeah, thanks :D